code/game/objects/structures/tables_racks.dm

```diff
@@ -37,6 +40,16 @@
 	var/buildstackamount = 1
 	var/framestackamount = 2
 	var/deconstruction_ready = TRUE
+	///Whether or not the table could be flipped or not
+	var/can_flip = TRUE
+	///Whether or not the table is flipped
+	var/is_flipped = FALSE
+	/// Whether or not when flipped, it ignores PASS_GLASS flag
+	var/is_transparent = FALSE
+	/// If you don't have sprites for flipped tables, you can use matrices instead. looks ever-slightly worse.
+	var/use_matrices_instead = FALSE
+	/// Matrix to return to on unflipping table
+	var/matrix/original_matrix
```

Comment:
Quick nitpick before we merge, please mention this regards being flipped in the var name